.github/workflows/windows-release-build.yml

```diff
@@ -0,0 +1,37 @@
+name: "Windows Release Build"
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  windows-release-build:
+    strategy:
+      fail-fast: false
+      matrix:
+        # Note: keep this in sync with the Windows matrix in tests.yml
+        php: [ "7.2", "7.3", "8.0", "8.1", "8.2" ]
```

Comment:
@mickverm: This would explain the missing PHP 7.4 builds.

@alcaeus: Can you make a ticket for this? We're missing 7.4 both here and in `tests.yml`.

I don't think it's possible to modify the action configs and re-run for 1.15.0, since the tag includes the config file omitting 7.4. Short of creating a 1.15.1 release, perhaps you can manually create 7.4 archives on your own fork (e.g. rewrite git history to create the 1.15.0 tag anew) and then attach those to the [1.15.0 release](https://github.com/mongodb/mongo-php-driver/releases/tag/1.15.0).

Reply:
Created PHPC-2185 to track this. I'll tag a 1.15.1 release soon to also include a fix for PHPC-2184, which will then provide 7.4 builds for the 1.15 branch.